SerialPrograms/Source/PokemonSV/Programs/Farming/PokemonSV_BlueberryCatchPhoto.h

```diff
@@ -31,7 +31,7 @@ void quest_photo(const ProgramInfo& info, ConsoleHandle& console, BotBaseContext
 void quest_catch_navi(const ProgramInfo& info, ConsoleHandle& console, BotBaseContext& context, const BBQOption& BBQ_OPTIONS, BBQuests current_quest);
 
 //Select and throw ball
-void quest_catch_throw_ball(const ProgramInfo& info, ConsoleHandle& console, BotBaseContext& context, Language language, std::string selected_ball);
+void quest_catch_throw_ball(const ProgramInfo& info, ConsoleHandle& console, BotBaseContext& context, Language language, const std::string selected_ball);
```

Comment:
const reference, not just const. Remember that we don't want to copy non-trivial objects unless we need to.